api/identifiers/serializers.py

```diff
@@ -65,8 +64,7 @@ def create(self, validated_data):
                 identifiers = get_or_create_identifiers(node)
             except HTTPError:
                 raise exceptions.ValidationError('Error response from client.')
-            except TypeError:
-                raise ServiceUnavailableError()
```

Comment:
Like the assumption that you couldn't mint DOIs locally, this apparently was just wrong.